crates/sui-graphql-client/src/lib.rs

```diff
@@ -262,18 +286,16 @@ impl Client {
 
     /// Get the list of active validators for the provided epoch, including related metadata.
     /// If no epoch is provided, it will return the active validators for the current epoch.
-    pub async fn active_validators(
+    pub async fn active_validators<'a>(
         &self,
         epoch: Option<u64>,
-        after: Option<String>,
-        before: Option<String>,
-        first: Option<i32>,
-        last: Option<i32>,
+        pagination_filter: Option<PaginationFilter<'a>>,
```

Comment:
Complementary to the question about `Option<Page<...>>` in the output, do we need the `Option` here? Seems like it would be just as easy for someone to pass `PaginationFilter::default()` as `None`, and it makes it clear what the parameter is.

Reply:
Sure, we can add that.